drivers/volume/gcp/gcp.go

```diff
@@ -506,6 +515,13 @@ func (g *gcp) GetRestoreStatus(restore *storkapi.ApplicationRestore) ([]*storkap
 		} else {
 			disk, err := g.service.Disks.Get(g.projectID, vInfo.Zones[0], vInfo.RestoreVolume).Do()
 			if err != nil {
+				if googleErr, ok := err.(*googleapi.Error); ok {
```

Comment:
You will need to ignore error in `GetRestoreStatus` call in application restore.go since we are processing volume information afterward

Reply:
ok so we are continue working in case of error

Reply:
With addressing @disrani-px comment, now it is handled properly.